### PR TITLE
fix #17: Adhere to Rust naming conventions

### LINKED ISSUE #17: Adhere to Rust naming conventions
We currently fully capitalize acronyms in type names. However, https://rust-lang-nursery.github.io/api-guidelines/naming.html specifies that in Rust the preference for acronyms is:

> In UpperCamelCase, acronyms and contractions of compound words count as one word: use Uuid rather than UUID, Usize rather than USize or Stdin rather than StdIn. In snake_case, acronyms and contractions are lower-cased: is_xid_start.

### PR DESCRIPTION
**Pull request by sssilver**

### CODE REVIEW

File: src/workerskv/mod.rs [hunk 1]
```diff
@@ -19,14 +19,14 @@ pub struct WorkersKVNamespace {
     pub title: String,
 }
 
-impl APIResult for WorkersKVNamespace {}
-impl APIResult for Vec<WorkersKVNamespace> {}
+impl ApiResult for WorkersKVNamespace {}
+impl ApiResult for Vec<WorkersKVNamespace> {}
```
Comment (ashleymichal) [inline src/workerskv/mod.rs]: Probably also need to do this for `WorkersKvNamespace`.

Comment (sssilver) [reply src/workerskv/mod.rs]: Yes! Good catch

Comment (sssilver) [reply src/workerskv/mod.rs]: @ashleymichal I can't find any occurrences of that in the current master which I've branched off of.

Comment (ashleymichal) [reply src/workerskv/mod.rs]: i mean `WorkersKVNamespace -> WorkersKvNamespace`, which is present here.

Comment (sssilver) [reply src/workerskv/mod.rs]: Done!

**Pull request merged**